### PR TITLE
Change minion's default port, enable minion health and readiness probe via helm chart

### PR DESCRIPTION
**Pull request by saurabhd336**
This PR adds support for minion's health and readiness probe when deploying using helm charts

### CODE REVIEW

**Comment by saurabhd336** [comment]
@Jackie-Jiang @daniellavoie Please have a look

File: pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java [hunk 1]
```diff
@@ -527,7 +527,7 @@ public static class Minion {
     public static final String CONFIG_OF_TASK_AUTH_TOKEN = "task.auth.token";
     public static final String CONFIG_OF_ADMIN_API_PORT = "pinot.minion.adminapi.port";
     public static final String MINION_TLS_PREFIX = "pinot.minion.tls";
-    public static final int DEFAULT_ADMIN_API_PORT = 6500;
+    public static final int DEFAULT_ADMIN_API_PORT = 9514;
```
[the remark this answers is not in the document]
Comment (saurabhd336) [reply pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java]: Ack.

File: pinot-tools/src/main/resources/conf/pinot-minion.conf [hunk 2]
```diff
@@ -33,7 +33,7 @@ pinot.set.instance.id.to.hostname=true
 # pinot.minion.host=localhost
 
 # Pinot Minion Port
-pinot.minion.port=8098
+pinot.minion.port=9154
 
 # Pinot Minion Admin API Port
 pinot.minion.adminapi.port=6500
```
Comment (Jackie-Jiang) [inline pinot-tools/src/main/resources/conf/pinot-minion.conf:39]: Remove this part

File: pinot-core/src/main/java/org/apache/pinot/core/util/ListenerConfigUtil.java [hunk 3]
```diff
@@ -155,23 +155,14 @@ public static List<ListenerConfig> buildServerAdminConfigs(PinotConfiguration se
   public static List<ListenerConfig> buildMinionAdminConfigs(PinotConfiguration minionConf) {
```
[the remark this answers is not in the document]
Comment (saurabhd336) [reply pinot-core/src/main/java/org/apache/pinot/core/util/ListenerConfigUtil.java:155]: Ack

File: pinot-tools/src/main/resources/conf/pinot-minion.conf [hunk 4]
```diff
@@ -33,7 +33,7 @@ pinot.set.instance.id.to.hostname=true
 # pinot.minion.host=localhost
 
 # Pinot Minion Port
-pinot.minion.port=8098
+pinot.minion.port=9154
```
[the remark this answers is not in the document]
Comment (saurabhd336) [reply pinot-tools/src/main/resources/conf/pinot-minion.conf:36]: Ack

File: pinot-tools/src/main/resources/conf/pinot-minion.conf (hunk 2 above)
Comment (saurabhd336) [reply pinot-tools/src/main/resources/conf/pinot-minion.conf:39]: Ack

**Pull request merged**